src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -123,6 +123,19 @@ public function testDontValidateIfParentWithoutCascadeValidation()
         $this->assertNoViolation();
     }
 
+    public function testNotExistingConstraintIndex()
+    {
+        $object = $this->getMock('\stdClass');
```

Comment:
Nope. I'd only copied cause of the lack of knowledge in writing tests. Thanks for the Hint!